stack-8.6.4.yaml

```diff
@@ -16,6 +16,7 @@ packages:
   - ./plugins/hls-explicit-imports-plugin
   - ./plugins/hls-refine-imports-plugin
   - ./plugins/hls-hlint-plugin
+  - ./plugins/hls-rename-plugin
```

Comment:
Since we don't expect the plugin to build with 8.6.x, should it be removed from this stack.yaml file?